conda/plugins/subcommands/doctor/__init__.py

```diff
@@ -35,10 +36,10 @@ def configure_parser(parser: ArgumentParser):
 
 
 def execute(args: argparse.Namespace) -> None:
-    """Run conda doctor subcommand."""
-    from .health_checks import display_health_checks
-
-    display_health_checks(context.target_prefix, verbose=context.verbose)
+    """Run registered health_check plugins."""
+    display_report_heading(context.target_prefix)
```

Comment:
This function is tiny, let's inline it and mark `display_report_heading` for deprecation.